### ISSUE TITLE
Notice and warning due to constants already defined

### ISSUE THREAD

**Issue by glueckpress** [intercom/intercom-wordpress#39]
When testing with any of `WP_DEBUG`, `WP_DEBUG_DISPLAY`, and/or `WP_DEBUG_LOG` manually set to `true` in _wp-config.php_, the plugin throws a notice and may cause other plugins to throw a warning.

> Notice: Constant WP_DEBUG_LOG already defined in /…/plugins/intercom/bootstrap.php on line 320
> Warning: session_start(): Cannot send session cache limiter - headers already sent (output started at /…/plugins/intercom/bootstrap.php:318) in {some-other-plugin}

Prepending a quick check with `if ( ! defined( '{CONSTANT}' ) )` to [lines 318-320 in bootstrap.php](https://github.com/intercom/intercom-wordpress/blob/master/bootstrap.php#L318-L320) solves the issue for me. However, I’m not too firm in automated testing, so apologies if this doesn’t come with a PR.

```php
if( ! defined( 'WP_DEBUG' ) )         define( 'WP_DEBUG', true );
if( ! defined( 'WP_DEBUG_DISPLAY' ) ) define( 'WP_DEBUG_DISPLAY', false );
if( ! defined( 'WP_DEBUG_LOG' ) )     define( 'WP_DEBUG_LOG', true );
```

**Comment by bobjflong** [comment]
Thanks, I think we can just delete those definitions, not sure they serve a purpose

**Comment by bobjflong** [comment]
Fixed in `2.2.3`

**Comment by glueckpress** [comment]
Thanks! :+1:
